pkg/console/util.go

```diff
@@ -160,6 +160,23 @@ func enableNTPServers(ntpServerList []string) error {
 	return nil
 }
 
+func updateDNSServersAndReloadNetConfig(dnsServerList []string) error {
+	dnsServers := strings.Join(dnsServerList, " ")
+	output, err := exec.Command("sed", "-i", fmt.Sprintf(`s/^NETCONFIG_DNS_STATIC_SERVERS.*/NETCONFIG_DNS_STATIC_SERVERS="%s"/`, dnsServers), "/etc/sysconfig/network/config").CombinedOutput()
+	if err != nil {
+		logrus.Error(err, string(output))
+		return err
+	}
+
+	output, err = exec.Command("netconfig", "update").CombinedOutput()
```

Comment:
NIT: would it be better running `netconfig update -m dns` that only updates DNS settings?